rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -646,6 +649,19 @@ class ProcMatcherSpec extends FlatSpec with Matchers {
     )
     result.knownFree should be(inputs.knownFree)
   }
+
+  it should "handle peek" in {
+    val basicInput = ParBuilder[Coeval]
+      .buildAST(
+        new StringReader("""for ( x, y <<- @Nil ) { x!(*y) }""")
```

Comment:
Get rid of the StringReader in here. Options:

1. add an overload in the ParBuilder object that takes a string (notice the ParBuilder trait has such an overload)

2. Make `buildAST` a util method in the InterpreterUtil, and hide the `Coeval` in it as well

Reply:
I'm going to do 3. which is add `buildAST(s: String)` to `ParBuilderUtil`